metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_usage_v2.py

```diff
@@ -411,6 +413,23 @@ def _get_operation_aspect_work_unit(
             operation_type = OPERATION_STATEMENT_TYPES.get(
                 query_type, OperationTypeClass.CUSTOM
             )
+            affected_rows = (
+                event.rows_inserted
```

Comment:
why can't we just sum all of the numbers together? do we need to make it all conditional based on the operation type?